content/en/monitors/monitor_types/process_check.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Process Check Monitor
+kind: documentation
+description: "Check if a process is running on a host"
+further_reading:
+- link: "monitors/notifications"
+  tag: "Documentation"
+  text: "Configure your monitor notifications"
+- link: "monitors/downtimes"
+  tag: "Documentation"
+  text: "Schedule a downtime to mute a monitor"
+- link: "monitors/monitor_status"
+  tag: "Documentation"
+  text: "Check your monitor status"
+---
+
+## Overview
+
+A process check monitor watches the status produced by `process.up` reported by the check in the Agent. At the Agent level you can [configure thresholds][1] based on the number of matching processes.
+
+## Monitor creation
+
+To create a [process check monitor][2] in Datadog, use the main navigation: *Monitors --> New Monitor --> Process Check*.
+
+### Pick a process
+
+From the drop-down list, select a process to monitor. Filter the list by entering search criteria.
+
+### Pick monitor scope
+
+Select the hosts to monitor by choosing host names, tags, or choose `All Monitored Hosts`.  Only hosts or tags reporting a status for the selected process are displayed. If you need to exclude certain hosts, use the second field to list names or tags.
```

Comment:
```suggestion
Select the hosts to monitor by choosing host names, tags, or choose `All Monitored Hosts`.  Only hosts or tags reporting a status for the selected process are displayed. If you need to exclude certain hosts, use the second field to list names or tags:
```

Reply:
I don't think a colon is appropriate here.